uncharted-aske/HMI
# Add mock data to provenance graph

**What** 

- Added mock data to provenance graph UI skeleton 
- **Update:** Added ProvenanceGraphInterface and change the Condensed | Expanded buttons to a toggle that shows the active layout

**How**

- Added `ProvenanceRenderer` to render operations as rectangles and model, parameters, and variables as ellipses

**Testing**

- Open Simulation View with one model and toggle the condensed and expanded buttons to see the single model workflow 
- Select two models and open simulation view to see the multi-model workflow 

**Screen Recording**

https://user-images.githubusercontent.com/23723378/129083716-540904f9-9ec5-4fb9-9291-9d5eb932e8b2.mov

**Updated Recording with Toggle**
https://user-images.githubusercontent.com/23723378/129400113-40098179-c7aa-4a2e-9d6d-fe413d76be0c.mov
